src/v2/Components/ArtistSeriesRail/__tests__/ArtistSeriesItem.jest.tsx

```diff
@@ -42,19 +42,23 @@ describe("Artist Series Rail Item", () => {
     const elem = component.find(RouterLink).first()
     elem.props().onClick({} as any)
 
-    expect(trackEvent).toHaveBeenCalledWith({
-      action: "clickedArtistSeriesGroup",
-      context_module: "context-module",
-      context_page_owner_id: "context-page-owner-id",
-      context_page_owner_slug: "context-page-owner-slug",
-      context_page_owner_type: "artist",
-      curation_boost: true,
-      destination_page_owner_id: "internal-id",
-      destination_page_owner_slug: "slug",
-      destination_page_owner_type: "artistSeries",
-      horizontal_slide_position: 2,
-      type: "thumbnail",
-    })
+    expect(trackEvent.mock.calls[0]).toMatchInlineSnapshot(`
+            Array [
+              Object {
+                "action": "clickedArtistSeriesGroup",
+                "context_module": "context-module",
+                "context_page_owner_id": "context-page-owner-id",
+                "context_page_owner_slug": "context-page-owner-slug",
+                "context_page_owner_type": "artist",
+                "curation_boost": true,
+                "destination_page_owner_id": "internal-id",
+                "destination_page_owner_slug": "slug",
+                "destination_page_owner_type": "artistSeries",
+                "horizontal_slide_position": 2,
+                "type": "thumbnail",
+              },
+            ]
+          `)
```

Comment:
same questions here as above

Reply:
in this case I just changed the way we assert on analytics from `toHaveBeenCalledWith` to -> `toMatchInlineSnapshot`.

The reason for that is that we discussed it a while ago with @pvinis and decided (as a result of [this rfc](https://github.com/artsy/cohesion/issues/212)) that it would be better to assert on analytics with `toMatchInlineSnapshot`

Reply:
oop yep you're totally right! sorry I missed this the first time around 😅